gateway/ln-gateway/src/lightning/ldk.rs

```diff
@@ -233,25 +286,14 @@ impl ILnRpcClient for GatewayLdkClient {
         }
         let node_status = self.node.status();
 
-        let Some(esplora_chain_tip_block_summary) = self
-            .esplora_client
-            .get_blocks(None)
-            .await
-            .map_err(|e| LightningRpcError::FailedToGetNodeInfo {
-                failure_reason: format!("Failed get chain tip block summary: {e:?}"),
-            })?
-            .into_iter()
-            .next()
-        else {
-            return Err(LightningRpcError::FailedToGetNodeInfo {
-                failure_reason:
-                    "Failed to get chain tip block summary (empty block list was returned)"
-                        .to_string(),
-            });
-        };
-
-        let esplora_chain_tip_block_height = esplora_chain_tip_block_summary.time.height;
-        let ldk_block_height: u32 = node_status.current_best_block.height;
+        let esplora_chain_tip_block_height =
+            self.bitcoind_rpc.get_block_count().await.map_err(|e| {
+                LightningRpcError::FailedToGetNodeInfo {
+                    failure_reason: format!("Failed to get block count from chain source: {e}"),
+                }
+            })? as u32
```

Comment:
As a routine we should avoid `as` casts, and use `TryFrom`

Reply:
Done.